src/Elastic.Apm/Api/Tracer.cs

```diff
@@ -206,5 +206,49 @@ public Task<T> CaptureTransaction<T>(string name, string type, Func<Task<T>> fun
 
 			transaction.End();
 		}, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
+
+		public void CaptureError(string message, string culprit, StackFrame[] frames = null, string parentId = null)
+		{
+			var capturedCulprit = string.IsNullOrEmpty(culprit) ? "PublicAPI-CaptureException" : culprit;
+
+			var capturedException = new CapturedException { Message = message };
+
+			if (frames != null)
+			{
+				capturedException.StackTrace
+					= StacktraceHelper.GenerateApmStackTrace(frames, _logger, _configProvider.CurrentSnapshot, _apmServerInfo,
+						"failed capturing stacktrace");
+			}
+
+			_sender.QueueError(new Error(capturedException, null, null, _logger) { Culprit = capturedCulprit });
+		}
+
+		public void CaptureException(Exception exception, string culprit = null, bool isHandled = false, string parentId = null)
+		{
+			var capturedCulprit = string.IsNullOrEmpty(culprit) ? "PublicAPI-CaptureException" : culprit;
```

Comment:
Same as above: should this check to see if there's a current transaction or current span here, and correlate the error with it?

Reply:
👍 fixed